test/parallel/test-http-write-empty-string.js

```diff
@@ -38,16 +38,16 @@ const server = http.createServer(function(request, response) {
   this.close();
 });
 
-server.listen(0, common.mustCall(function() {
-  http.get({ port: this.address().port }, common.mustCall(function(res) {
+server.listen(0, common.mustCall(() => {
+  http.get({ port: this.address().port }, common.mustCall((res) => {
```

Comment:
arrow functions bind `this` to its top level scope. so please replace `this.address()` with `server.address()`

Reply:
Good catch! I was a bit sloppy here — [L38](https://github.com/nodejs/node/pull/24483/files#diff-9cd31a3484f2994575e6dd2f94b43ea1R38) is another spot I missed.